module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/ReviewsTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * HoldingsILS Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2019.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\RecordTab;
+
+use VuFind\RecordTab\Reviews;
+
+/**
+ * HoldingsILS Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+
+class ReviewsTest extends \PHPUnit\Framework\TestCase
+{
+    Public function testGetDescription()
```

Comment:
In our code style, keywords are lowercase, so I would change `Public` to `public`.

Reply:
We also put PHPDoc comments on every function, so I'd suggest adding something like:

```
/**
 * Test getting description.
 *
 * @return void
 */
```

Reply:
Finally, while types are optional in PHP, I find it helpful to include typehints in tests to make things as explicit as possible (I expect that in future, we'll be more strongly typing all of VuFind's code, since this is a direction the PHP language seems to be heading in). So you could add `: void` to the end of the function definition to indicate the void return type.